unity-client/Assets/Scripts/MainScripts/DCL/Controllers/StickersController/StickersController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+public class StickersController : MonoBehaviour
```

Comment:
Why do we need a specialized stickers controller? Why we can't abstract this out with the rest of our expressions?

Reply:
Mostly because our expressions/emotes are not in a controller, they are just triggered by our `AvatarAnimator` script.
I also want to decouple this system from the expressions due to one of the requierements specified here: https://app.zenhub.com/workspaces/unity-6047bad476c3c0001942ee91/issues/decentraland/unity-renderer/9

We will need to trigger stickers outside of the emotes as reactions through the chat or a new UI, so decoupling this right now will help us in the future.

Reply:
I think I still don't fully understand the implications of the "sticker" emotes. I see that issue and for me its just another emote but with triggered particles. Expressions can be triggered from chat and UI right now, so I don't see the true difference.

Reply:
So we give more flexibility to the users to express themselves. They can wave or dance while triggering a heart or some clapping stickers. They could combine stickers and emotesin any way they want.
It has the advantage of not overriding the current animation  so if you want to show some stickers, you trigger them while running, or jumping or doing whatever. 

And also the system allows our emotes to throw stickers if we want through AnimationEvents so it doesnt collide with anything of the above.